docs/getting_started.rst

```diff
@@ -153,14 +153,10 @@ download them using the downloader cog.
 
 You can start using the downloader cog by loading it: ``[p]load downloader``
 
-You can find cogs by searching on ``cogs.red``. Find whatever you want,
+You can find cogs by searching on `<https://index.discord.red>`_. Find whatever you want,
 there are hundreds of cogs available!
 
-.. note:: ``cogs.red``, the website that list all of the cogs is not
-    ready for v3 yet. For now, you can refer to `this post
-    <https://cogboard.red/t/approved-repositories/210>`_.
-
-.. 26-cogs not available, let's use my repo :3
+.. note:: We're currently working on a better website for listing repositories and their cogs. Stay tuned!
```

Comment:
```suggestion
.. note:: A even better way to discover new cogs and repositories is in the works! Stay tuned!
```
I know that it wasn't the intention but the previous message seemed to imply that the current one is bad :')

Reply:
Thanks, I had troubles wording that one :)